lib/widgets/carousel.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:flutter/material.dart';
+
+final List<String> imgList = [
+  'https://sochem.in/static/media/slide-home-3.d001fcc5.jpg',
+  'https://osmoze.in/assets/images/gallery/2.png',
+  'https://iitbhu.ac.in/sites/default/files/styles/iitbhu_slides/public/slides/che/slide_che_02.jpg?itok=R1Cmzwfi',
```

Comment:
I suppose using this links here is just temporary, after we connect it to backend we would be fetching the urls for images from database. Correct me if I am wrong

Reply:
Yes, these are temporary links just for a glimpse of the actual app when ready. With required backend additions, these will supposedly use necessary pictures.